src/blocknative.rs

```diff
@@ -283,7 +285,7 @@ mod tests {
           "blockPrices": [
             {
               "blockNumber": "13005096",
```

Comment:
When I checked the BlockNative gas estimator, it looks like their API returns JSON numbers (and not strings) for all the numerical values:
```
$ curl -s https://blocknative-api.herokuapp.com/data | jq
{
  "pendingBlockNumberVal": 13313526,
  "seconds": 2,
  "maxPrice": 252,
  "estimatedTransactions": 104,
  "estimatedPrices": [
    {
      "confidence": 99,
      "price": 56,
      "maxPriorityFeePerGas": 1.99,
      "maxFeePerGas": 111.21
    },
    {
      "confidence": 95,
      "price": 56,
      "maxPriorityFeePerGas": 1.94,
      "maxFeePerGas": 111.15
    },
    {
      "confidence": 90,
      "price": 56,
      "maxPriorityFeePerGas": 1.89,
      "maxFeePerGas": 111.1
    },
    {
      "confidence": 80,
      "price": 56,
      "maxPriorityFeePerGas": 1.79,
      "maxFeePerGas": 111
    },
    {
      "confidence": 70,
      "price": 56,
      "maxPriorityFeePerGas": 1.69,
      "maxFeePerGas": 110.9
    }
  ],
  "baseFeePerGas": 54.605943748
}
```

Don't know if this uses a different API or not, or if it has different format than the one that they use on https://www.blocknative.com/gas-estimator. I don't have an API key so I wasn't able to test :disappointed:

Reply:
True. UT was passing since those values were not used. Fixed.